samples/sample_common/src/sample_utils.cpp

```diff
@@ -1019,7 +1019,7 @@ mfxStatus CSmplYUVWriter::WriteNextFrame(mfxFrameSurface1 *pSurface)
     case MFX_FOURCC_P010:
     case MFX_FOURCC_P210:
     {
-        mfxU32 height = pInfo.FourCC == MFX_FOURCC_P210 ? (mfxU32)pInfo.CropH : (mfxU32)pInfo.CropH / 2;
+        mfxU32 height = pInfo.FourCC == MFX_FOURCC_P210 ? (mfxU32)pInfo.CropH : (mfxU32)(pInfo.CropH + 1) / 2;
```

Comment:
+1? that's something not obvious...

Reply:
discussed offline. That's ok since we have CropH here, not Height which should be multiple of 2.

Reply:
Yes. I find some VP9 P010 dynamic resoluion video clip has odd cropping height. So UV height will be wrong. Thanks.

Reply:
I think we likely need similar fix for other color formats where we have /2. Or we need to reconsider the change and require crops to be aligned by 2. Anyhow, this is a good observation, but I feel we need more work here. For this reason, -1 to avoid merging in partial fix.

Reply:
Sure. I will check the other color format and refine this PR. Thanks.